projects/kit/src/lib/masks/date-time/tests/date-time-separator.spec.ts

```diff
@@ -0,0 +1,49 @@
+import {MASKITO_DEFAULT_OPTIONS, maskitoTransform} from '@maskito/core';
+
+import type {MaskitoTimeMode} from '../../../types';
+import {maskitoDateTimeOptionsGenerator} from '../date-time-mask';
+
+describe('DateTime | dateTimeSeparator', () => {
+    const dateTimeSeparators = [':', ';_', '_-_', '_at_'];
+    let options = MASKITO_DEFAULT_OPTIONS;
+
+    dateTimeSeparators.forEach(dateTimeSeparator => {
+        const testCases: Array<{
+            typedDigits: string;
+            formattedValue: string;
+            timeMode: MaskitoTimeMode;
+        }> = [
+            {
+                typedDigits: '050220040341',
+                formattedValue: `05.02.2004${dateTimeSeparator}03:41`,
```

Comment:
This birthday date will be always here now :D

Reply:
Hehe))